File: example-java/src/test/java/com/willmolloy/HelloWorldTest.java
package com.willmolloy;

import static com.google.common.truth.Truth.assertThat;

import org.junit.jupiter.api.Test;

/**
 * Unit tests for {@link HelloWorld}.
 *
 * @author <a href=https://willmolloy.com>Will Molloy</a>
 */
class HelloWorldTest {

  @Test
  void test_hello() {
    assertThat(new HelloWorld().hello("world")).isEqualTo("Hello world!");
  }
}

File: example-kotlin/src/test/kotlin/com/willmolloy/HelloWorldTest.kt
package com.willmolloy

import com.google.common.truth.Truth.assertThat
import org.junit.jupiter.api.Test

/**
 * Unit tests for [HelloWorld].
 */
class HelloWorldTest {
  @Test
  fun `test hello`() {
    assertThat(HelloWorld().hello("world")).isEqualTo("Hello world!")
  }
}
